admin/src/tables/GeneratorShortcodeTable.jsx

```diff
@@ -24,6 +24,7 @@ import {
 import useTableUpdater from '../hooks/useTableUpdater';
 import useChangeRow from '../hooks/useChangeRow';
 import useTablePanels from '../hooks/useTablePanels';
+import {useState} from "react";
```

Comment:
react functions should always be imported first (not a bug, just a way how we write it)